server/lib/cloudflare.js

```diff
@@ -0,0 +1,44 @@
+import config from 'config';
+import cloudflare from 'cloudflare';
+
+import logger from './logger';
+
+// Load config
+const cfConfig = config.cloudflare || {};
+
+// Check config
+const isLiveServer = ['staging', 'production'].includes(config.env);
+const hasConfig = Boolean(cfConfig.email && cfConfig.key && cfConfig.zone);
+
+if (isLiveServer && !hasConfig) {
+  logger.warn('Your Cloudflare config is imcomplete, you must provide an email, a key and a zone');
```

Comment:
Same comment than the other day about "your", don't think we should formulate errors/warnings like that.